en/docs/Learn/APIMicrogateway/microgateway-quick-start.md

```diff
@@ -14,7 +14,7 @@ Microgateway offers you a proxy that is capable of performing security validatio
 Before you begin,
 
 1.  Download the [Microgateway](https://wso2.com/api-management/api-microgateway/) distribution and extract it.
-2.  Append the full path of the `           /bin          ` folder of the extracted **** Microgateway distribution to the PATH environment variable.
+2.  Append the full path of the `           /bin          ` folder of the extracted Microgateway distribution to the PATH environment variable.
```

Comment:
Shouldn't we remove the MGW related content from the API-M 3.0.0 docs (en/docs/Learn/APIMicrogateway/)  and point off to [1] with regard to MGW content, because the MGW 3.0.1 docs are newer than this?

[1] https://docs.wso2.com/display/MG301/API+Microgateway+Documentation

Reply:
Yes this will be removed. So we can ignore this.